docs/processing_information.md

```diff
@@ -43,7 +43,7 @@ For single-nuclei samples, all counts for spliced cDNA and intronic regions were
 
 #### Filtering cells
 
-We then removed any cell barcodes that are likely to correspond to empty droplets, rather than true cells. 
+In addition to an unfiltered counts matrix, we provide a matrix filtered to only droplets that are likely to include cells, removing any cell barcodes that are likely to correspond to empty droplets.
```

Comment:
The second clause seemed redundant to me, but including "cell barcodes" seems important, so what do you think of this?
```suggestion
In addition to an unfiltered counts matrix, we provide a matrix filtered to only cell barcodes from droplets that are likely to include true cells.
```

Reply:
I like it! Will go ahead and commit that change.